app/src/main/res/values/strings.xml

```diff
@@ -52,6 +52,7 @@
     <string name="mark_as_read">Mark as Read</string>
     <string name="mark_as_unread">Mark as Unread</string>
     <string name="me">Me</string>
+    <string name="delete_messages">Delete message</string>
```

Comment:
lets just use Delete, so we dont need new strings. And even if you add a new string, it has to be added into all language files.